packages/anasazi/src/AnasaziTraceMinRitzOp.hpp

```diff
@@ -89,7 +105,7 @@ template <class Scalar, class MV, class OP>
 class TraceMinProjOp
 {
   typedef Anasazi::MultiVecTraits<Scalar,MV>    MVT;
-  const Scalar ONE; 
```

Comment:
If you actually did the Anasazi build and tests, I'm not so worried about the minor changes to source.  However, it would be cleaner to have comments-only changes, so if you're in the middle of that, I'll be happy if you finish :-)